tests/phpunit/tests/test-rest-request.php

```diff
@@ -142,6 +146,34 @@ public function test_should_not_define_language_when_not_sent( $data ) {
 		$this->assertNull( $this->pll_rest->curlang );
 	}
 
+	/**
+	 * @testWith [{"type": "post", "route": "/wp/v2/posts", "requiredField": "title", "fieldValue": "Post"}]
+	 *           [{"type": "term", "route": "/wp/v2/categories", "requiredField": "name", "fieldValue": "Category term"}]
+	 *
+	 * @param array $data {
+	 *    @type string $type          Type of content.
+	 *    @type string $route         Route.
+	 *    @type string $requiredField Required field to create the content.
+	 *    @type string $fieldValue    Value of the required field.
+	 * }
+	 */
+	public function test_should_assign_default_language_when_no_language_sent( $data ) {
```

Comment:
It would be more readable with 4 parameters instead of an array:
```suggestion
	public function test_should_assign_default_language_when_no_language_sent( $type, $route, $field, $value ) {
```

Reply:
Done in b25e4420edfd387f9bcdfeb9242d66e13add143f